code/modules/surgery/organs/stomach/stomach_ethereal.dm

```diff
@@ -48,24 +49,32 @@
 /obj/item/organ/stomach/ethereal/proc/handle_charge(mob/living/carbon/carbon, delta_time, times_fired)
 	switch(crystal_charge)
 		if(-INFINITY to ETHEREAL_CHARGE_NONE)
+			SEND_SIGNAL(carbon, COMSIG_ADD_MOOD_EVENT, "charge", /datum/mood_event/decharged)
```

Comment:
Mood code makes me depressed

Reply:
it's possible but it's just how moodlets should be done in general

every moodlet should be essentially a representation of a signal handler thats handled in the mood component itself

Reply:
ghhhhhhhhhhhhh